src/styles/blocks/mixin.scss

```diff
@@ -0,0 +1,13 @@
+@mixin hoverEffect {
+  cursor: pointer;
+  background-color: #ffc0cb;
+  transform: translateY(-20px);
+}
+
+@mixin calendarNumber {
```

Comment:
These mixins are redundant as they useful only in specific cases

Reply:
Sorry! I understood wrong @nesting. I thought that you mentioned about @mixin. The brain doesn't work.  Thank you.